packages/@lwc/engine-server/src/apis/render-component.ts

```diff
@@ -30,8 +30,9 @@ const FakeRootElement: HostElement = {
 export function renderComponent(
     tagName: string,
     Ctor: typeof LightningElement,
-    props: { [name: string]: any } = {}
-): string {
+    props: { [name: string]: any } = {},
+    options: { syntheticShadow?: boolean } = {}
```

Comment:
ins't this information already exosed via `renderer`

Reply:
Yes, it is exposed, but on SSR we [default to `false`](https://github.com/salesforce/lwc/pull/2093/files#diff-eec7b351db424f272f192870d35a1ed36d4fe2950c2085feb9c0d2fea5d0f25fL65) so far. This PR allows configuring the renderer `syntheticShadow` value.